homework/PavelBo/loops/userLoginService/src/main/java/com/tutrit/java/quickstart/UserLoginService.java

```diff
@@ -0,0 +1,36 @@
+package com.tutrit.java.quickstart;
+
+public class UserLoginService {
+
+    User user;
+
+    public UserLoginService(User user){
+        this.user = user;
+    }
+
+    public boolean login(User user, String password) {
+        if (user.isBLOCKED()){
+            return false;
+        }else {
+            if (isPassword(user, password)){
+                user.resetAttempts();
+                return true;
+            }else {
+                user.setАttempts(user.getАttempts() - 1);
+                blockedUser(user);
```

Comment:
`blockedUser(user);` -> `blockUser(user);`will be more readable

also, I suggest that you use Atl+Ctrl +L before merge